packages/dd-trace/src/opentracing/span.js

```diff
@@ -97,7 +97,7 @@ class DatadogSpan {
       unfinishedRegistry.register(this, operationName, this)
     }
     spanleak.addSpan(this, operationName)
-    startCh.publish(this)
+    startCh.publish({ span: this, fields })
```

Comment:
Now that we are creating a new object, we should check `startCh.hasSubscribers` before sending the event, to prevent creating new objects when it is not necessary.

Reply:
oops, true